nginx/docker/confd/templates/app.conf.tmpl

```diff
@@ -14,6 +14,11 @@ server {
     access_log  /var/log/app/nginx.access.json logstash_json;
     error_log   /var/log/app/nginx.error.log error;
 
+    {{ if exists "/opg/nginx/clientmaxbodysize" }}
```

Comment:
We don't need an else section as it will just default to what is in the nginx.conf